static/app/views/releases/detail/commitsAndFiles/filesChanged.tsx

```diff
@@ -74,13 +74,15 @@ function FilesChanged({activeReleaseRepo, releaseRepos, projectSlug}: FilesChang
         )}
       />
       <Layout.Body>
-        <Layout.Main fullWidth>
-          {releaseRepos.length > 1 && (
+        {releaseRepos.length > 1 && (
+          <Layout.Main fullWidth>
```

Comment:
Did you mean to add a second `Layout.Main` here?

Reply:
yeah using it for the padding
 
this is how prod looks
![image](https://github.com/user-attachments/assets/3d7c6ff9-8596-4d50-829b-57b68491b3ff)